code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -287,7 +287,7 @@
 				missing_organs += "lungs"
 			if(!(TRAIT_LIVERLESS_METABOLISM in the_dudes_species.inherent_traits) && !humantarget.get_organ_slot(ORGAN_SLOT_LIVER))
 				missing_organs += "liver"
-			if(the_dudes_species.mutantstomach && !humantarget.get_organ_slot(ORGAN_SLOT_STOMACH))
+			if(!(TRAIT_NOHUNGER in the_dudes_species.inherent_traits) && !humantarget.get_organ_slot(ORGAN_SLOT_STOMACH))
```

Comment:
Not immediately relevant but should these maybe be replaced with `!HAS_TRAIT_FROM(TRAIT_X, the_dude, SPECIES_TRAIT)`?

Checking `in inherent_traits` seems redundant for mobs which are already that species

Reply:
Probably a good idea?